src/views/users/Users.jsx

```diff
@@ -297,15 +296,26 @@ class Users extends React.Component {
       }
     ];
   }
+
   //User's Vehicle List
-  userVechileFun(e) {
-    var user_id = parseInt(e.currentTarget.dataset.user_id);
-    var data = { UserId: user_id };
-    this.props.getUserVehicleDetails(data);
+  expand_row(row) {
+    var expanded = { ...this.state.expanded };
```

Comment:
Expanded is initialized as null, cant spread null value error is possible.

Destructure state and assign empty object as default value

Reply:
Fixed